### DISCUSSION TITLE
Let's Encrypt SSL - [MASTER] error: certificate has expired

### DISCUSSION THREAD

**Discussion by mcmenkes** [requarks/wiki#3463] [Help / Questions]
Hi, 

first of all thanks you very much for this great piece of Wiki ;-) 

I have trouble setting up the the Let's Encrypt SSL certificate following this tutorial: https://docs.requarks.io/install/ubuntu

DNS entry is setup and working. HTTP and HTTPS is passing the firewall to the WikiJS machine. WikiJS is fully working with HTTP. 
But if follow the steps _'Automatic HTTPS with Let's Encrypt_ (optional)'_ and 

`docker start wiki`
`docker logs wiki`

I get the following output: 

`[MASTER] info: =======================================
[MASTER] info: = Wiki.js 2.5.159 =====================
[MASTER] info: =======================================
[MASTER] info: Initializing...
 [MASTER] info: Using database driver pg for postgres [ OK ]
 [MASTER] info: Connecting to database...
 [MASTER] info: Database Connection Successful [ OK ]
[MASTER] info: Loading GraphQL Schema...
[MASTER] info: GraphQL Schema: [ OK ]
[MASTER] info: HTTP Server on port: [ 3000 ]
[MASTER] info: (LETSENCRYPT) Initializing Let's Encrypt client...
[MASTER] info: HTTP Server: [ RUNNING ]
[MASTER] info: (LETSENCRYPT) Generating server keypairs...
[MASTER] info: (LETSENCRYPT) Server keypairs generated successfully [ OK ]
[MASTER] info: (LETSENCRYPT) Generating certificate signing request (CSR)...
[MASTER] info: (LETSENCRYPT) CSR generated successfully [ OK ]
[MASTER] info: (LETSENCRYPT) Requesting certificate from Let's Encrypt...
[MASTER] info: (LETSENCRYPT) Setting HTTP challenge for mywiki.domain.com: [ READY ]
[MASTER] info: (LETSENCRYPT) Waiting for challenge to complete...
[MASTER] info: (LETSENCRYPT) Removing HTTP challenge: [ OK ]
[MASTER] info: (LETSENCRYPT) Removing HTTP challenge: [ OK ]
[MASTER] warn: (LETSENCRYPT) Error: certificate has expired
[MASTER] error: certificate has expired`

WikiJS is not working then. 

**Why _expired_**, if i want a to request a new one?
Any ideas?

Thanks in advance, 

MCm

**Host Info**
OS: Ubuntu 20.04
Wiki.js version:  2.5.159
Database engine: postgres, newest

**Comment by Xaver106** [comment]
Had the same problem. Just restarting the server fixed it for me.
